reporters/reporter-default/src/main/java/org/radargun/reporting/html/HtmlReporter.java

```diff
@@ -93,14 +105,19 @@ private void writeNormalizedConfigDocuments(Collection<Report> reports) {
             }
             for (String config : normalized) {
                NormalizedConfigDocument document = new NormalizedConfigDocument(
-                     targetDir, report.getConfiguration().name, setup.group, report.getCluster(), config, report.getNormalizedServiceConfigs(), slaves);
+                       targetDir, report.getConfiguration().name, setup.group, report.getCluster(), config, report.getNormalizedServiceConfigs(), slaves);
+               freemarker.template.Configuration cfg = initConfig();
                try {
-                  document.open();
-                  document.writeProperties();
-               } catch (IOException e) {
-                  log.error("Failed to write normalized configuration", e);
-               } finally {
-                  document.close();
+                  document.createReportDirectory();
+                  Template reportTemplate = cfg.getTemplate("normalizedReport.ftl");
+                  PrintWriter printWriter = new PrintWriter(targetDir + File.separator + document.getFileName());
+
+                  Map root = new HashMap();
+                  root.put("normalized", document);
+                  reportTemplate.process(root, printWriter);
+                  printWriter.close();
```

Comment:
Resources should be closed in finally {} block. Preferably use try-with-resources construct.